### ISSUE TITLE
Connect to Local Gitlab not working

### ISSUE THREAD

**Issue by ad3n** [Apicurio/apicurio-studio#1801]
I read https://www.apicur.io/studio/docs/using-a-local-installation-of-gitlab but when add new Identity Provider, UI is different with the doc

<img width="973" alt="Screen Shot 2022-01-31 at 12 13 05" src="https://user-images.githubusercontent.com/7464920/151742356-9174c7af-fd3f-431c-ad27-a1d6abaa2bec.png">

How to fill "Client Authentication" and "Client Assertion Signature Algorithm", i try to change the value randomly but always got error

Thank you for advice

**Comment by EricWittmann** [comment]
Oh I don't know the answer to this.  Clearly what has happened is that Keycloak has been updated since I wrote that documentation.  We may need to look for help in the Keycloak docs.

@carlesarnal any insight into this?

**Comment by DyspC** [comment]
In keycloak 12-15, gitlab account linking works with the client authentication set as "send client secret as post"

**Comment by ad3n** [comment]
@DyspC Thank you very much that work for me

**Comment by EricWittmann** [comment]
Any volunteers to update the document/article we have on this?  Or maybe just explain to me (e.g. in this GH issue) what docs change is needed and I"ll do it? :) :)

**Comment by EricWittmann** [comment]
ELI5 :)

**Comment by DyspC** [comment]
Updating the first screenshot of https://www.apicur.io/studio/docs/using-a-local-installation-of-gitlab to show the client authentication field and its value should be enough imo

![image](https://user-images.githubusercontent.com/17125930/152570294-ea9691dc-3513-4652-b2fe-bad424f3bf1e.png)

**Comment by EricWittmann** [comment]
Perfect, thanks!
